.pipelines/azure-pipeline-build.yml

```diff
@@ -280,9 +323,36 @@ jobs:
       inputs:
         scanType: 'Register'
         verbosity: 'Verbose'
-        dockerImagesToScan: '$(WINDOWS_FULL_IMAGE_NAME)'
+        dockerImagesToScan: '$(WINDOWS_FULL_IMAGE_NAME)-$(WINDOWS_2022_BASE_IMAGE_VERSION)'
         alertWarningLevel: 'High'
 
+- job: WindowsMultiArch
+  displayName: "Build windows multi-arch image"
+  pool:
+    name: Azure-Pipelines-Windows-CI-Test-EO
+  dependsOn:
+  - Common
+  - Windows2019
+  - Windows2022
+  variables:
+    WINDOWS_FULL_IMAGE_NAME: $[ dependencies.common.outputs['setup.WINDOWS_FULL_IMAGE_NAME'] ]
```

Comment:
kaveesh, because the deployed multi-arch image tag will be the same for both 19 & 22, so we have telemetry on whether the node is 19 or 22 ?

Reply:
have updated telegraf. Its now collecting a windowsVersion variable which tells us if its a 2019 or a 2022 node